kernel/packages/scene-system/scene.system.ts

```diff
@@ -516,6 +518,9 @@ export default class GamekitScene extends Script {
       }
 
       try {
+        sceneIsStatic = !(source.indexOf('ISystem') !== -1 || source.indexOf('OnPointer') !== -1)
```

Comment:
This is super hacky hahaha.

I know this is a test, but if we decide to try and detect static scenes, let's see if we can do better. I'm not sure this is a reliable test to ensure a scene is static or not.